src/main/ui/src/components/SearchByState/SearchByState.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {useState} from 'react';
+import ParkData from './data.json';
+
+export default function SearchByState() {
+    const [data, setData] = React.useState([]);
+    const [selectState, setSelect] = useState('');
+     React.useEffect(() => {
+     let url = `https://developer.nps.gov/api/v1/parks?parkCode="all"&limit=469&api_key=JMZizGv6gAcjBzoD4TbqW9RQSe9K8fEt9Cdb2Zta`;
+     fetch(url)
+         .then((response) => response.json())
+         .then((json) => {
+             setData(json.data);
+         })
+     .catch((error) => console.log(error));
+ }, [selectState]);
+
+     if (data.length > 0) {
+         data.filter((i) => {
```

Comment:
please use a more descriptive variable name `i` doesn't really let me know what you are iterating through.